examples/basic-typescript/package.json

```diff
@@ -8,7 +8,7 @@
     "icons": "npx @svgr/cli --typescript --replace-attr-values \"#000=currentColor\" --out-dir generated/icons -- public/icons"
   },
   "dependencies": {
-    "@makeswift/runtime": "^0.23.8",
+    "@makeswift/runtime": "^0.23.9",
```

Comment:
Bump `runtime` to include the latest [`Link` prop controller fix](https://github.com/makeswift/makeswift/releases/tag/%40makeswift%2Fruntime%400.23.9).